src/components/Select/index.js

```diff
@@ -115,43 +115,28 @@ export const Select = React.forwardRef(function Select(
               'contacto-multi-select-modal',
               maxTagModalTitle ? '' : 'contacto-multi-select-modal--no-title',
             ].join(' ')}
-            title={maxTagModalTitle}
+            title={'Hello'}
```

Comment:
Is it always going to be hello?

Reply:
No, Sorry
That was a debug statement
Lemme revert